app/README.md

```diff
@@ -2,35 +2,74 @@
 
 The Twilio Voice React Native Reference App is an example implementation of the Twilio Voice React Native SDK and serves to inspire developers who want to leverage the power of Twilio Programmable Voice in their React Native applications. See the Twilio Voice React Native SDK [here](https://github.com/twilio/twilio-voice-react-native).
 
+### Setting up Auth0
+
+Find the `Domain` and `Client ID` found in the `Settings` tab of your Auth0 Application
+
+![auth0nativesettings](https://user-images.githubusercontent.com/35968892/226489462-1236918f-d1e0-4c9b-944a-1d6175a862ab.png)
```

Comment:
Instead of masking out the `Domain` in this screenshot, try putting a test domain such e.g. mydomain.com. If you don't want to show the real value here, you can "inspect" the element and change the value from the inspector then take a screenshot. That way, the customer knows what a real value look like.